python/lib/cloudutils/syscfg.py

```diff
@@ -193,6 +200,17 @@ def __init__(self, glbEnv):
                          nfsConfig(self),
                          cloudAgentConfig(self)]
 
+#it covers RHEL8
```

Comment:
Remove comment

Reply:
@rhtyd was just following the norm in the file which has a comment for rest of the distro :). Will remove if needed